srtcore/epoll.cpp

```diff
@@ -91,7 +91,7 @@ CEPoll::~CEPoll()
 {
 }
 
-int CEPoll::create()
+int CEPoll::create(CEPollDesc** pout)
```

Comment:
This is necessary for the socket groups to get also direct access to the Desc structure, necessary to use the `swait` (new) function.